Mage/src/main/java/mage/game/command/Emblem.java

```diff
@@ -77,6 +77,23 @@ public void setSourceObject(MageObject sourceObject) {
         }
     }
 
+    /**
+     * For attributing a fake source from an xmage token.
+     */
+    public void setSourceObjectXmage(String xmageKey) {
```

Comment:
That's code setup only image, so must be named same way. I recommends to move it to rad emblem.

Reply:
makes sense.